packages/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -113,8 +113,10 @@ public function get_css( $should_prettify = false, $indent_count = 0 ) {
 		$declarations_indent = $should_prettify ? $indent_count + 1 : 0;
 		$suffix              = $should_prettify ? "\n" : '';
 		$spacer              = $should_prettify ? ' ' : '';
-		$selector            = $should_prettify ? str_replace( ',', ",\n", $this->get_selector() ) : $this->get_selector();
-		$css_declarations    = $this->declarations->get_declarations_string( $should_prettify, $declarations_indent );
+		// Trims any multiple selectors strings.
+		$selector_trimmed = implode( ',', array_map( 'trim', explode( ',', $this->get_selector() ) ) );
+		$selector         = $should_prettify ? str_replace( array( ',' ), ",\n", $selector_trimmed ) : $selector_trimmed;
+		$css_declarations = $this->declarations->get_declarations_string( $should_prettify, $declarations_indent );
```

Comment:
Since this adds an extra step of processing, what do you think about only enabling this when `$should_prettify` is `true`?

Reply:
Thanks for looking at this. 

That's a fair point, and I totally agree. 

Just to note that if we skip it, it'll mean that non-prettified selectors will have the space, e.g., the difference between:

`.test, .me{...}`

and

`.test,.me{...}`

This is, as they say, "no big deal". 😄

Reply:
DONE!